configs/classification/resnet/cifar_resnet50.py

```diff
@@ -1,15 +1,9 @@
-_base_ = '../../base.py'
+_base_ = './resnet50_b32x8_100e_jpg.py'
```

Comment:
1. cifar10 training from scratch may not need 8 gpus
2. if use only download this configfile, can we use it to train successfully

Reply:
only reuse model and eval_config